### DISCUSSION TITLE
Feature request: Server side non transcational write support

### DISCUSSION THREAD

**Discussion by grizzlybearer** [openfga/community#306] [Ideas]
OpenFGA currently supports non transactional writes on the [SDK side](https://github.com/openfga/python-sdk/tree/main#non-transaction-mode), by making a sequential write call per item, which is inefficient for our use-case for frequently writing >100k , >500k tuples.

It would be greatly beneficial if the Write API in the OpenFGA server itself supported this non transactional write mode.

A huge pain point we have with this is needing to read all the tuples of the store upfront, to compute the diff of pre-existing tuples.
We had to introduce a Postgres Read replica for this.

**Comment by Vic-Dev** [comment]
Duplicated of https://github.com/openfga/community/issues/265.
